README.md

```diff
@@ -18,3 +18,44 @@ usage: run_experiment.py [-h] [-j THREADS] experiment_yaml
 run_experiment.py: error: the following arguments are required: experiment_yaml
 ```
 
+**How to Install BFASST** 
+
+* Clone the github repository into your home directory in linux. Repository is found at https://github.com/byuccl/bfasst. Use ```git clone https://github.com/byuccl/bfasst.git.``` (Note: if Git is not installed on your device, use ```sudo apt install git``` first.) 
+
+* Use ```git pull``` to put the updated repo onto your device. 
+
+* Open a terminal in Visual Studio. Use ```cd bfasst``` if you're not in the bfasst folder already. (It should currently say _YOURNAME@____ _:~/bfasst$_)
+
+* Set up your SSH with caedm. You'll know if you set it up correctly when you can use ```ssh caedm``` and log in without having to enter your password. Follow the guide at https://byu-cpe.github.io/ComputingBootCamp/tutorials/linux/ .
```

Comment:
Can you make this a link?  Also, this is only required if you are running conformal on the CAEDM servers, so this should be indicated as an optional step.

Reply:
Do you mean turn the entire step into the link at the end?